### PR TITLE
Enhance restart required message trigger

### PR DESCRIPTION
**Pull request by luis-pereira**
WIP for fixing https://github.com/lxde/lxqt/issues/840

### CODE REVIEW

**Comment by paulolieuthier** [comment]
GTM.

**Comment by jleclanche** [comment]
lgtm. what's still needed?

**Comment by luis-pereira** [comment]
@jleclanche The autostart page and the LXQt modules in the basic page.

**Comment by luis-pereira** [comment]
@palinek @paulolieuthier @jleclanche  Finished.

**Comment by jleclanche** [comment]
gtm

**Comment by palinek** [comment]
GTM

... but the wording of the message is IMO a bit misleading... "You need to restart desktop session (lxqt-session) to reload settings. Use logout from the main menu"

to reload settings => to use the settings

File: lxqt-config-session/autostartutils.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA
+ */
+
+#ifndef AUTOSTARTUTILS_H
+#define AUTOSTARTUTILS_H
+
+#include <XdgDesktopFile>
```
Comment (palinek) [inline lxqt-config-session/autostartutils.h]: No need to include the whole XdgDesktopFile here... the forward declaration of `XdgDesktopFile` is enough, not?

Comment (luis-pereira) [inline lxqt-config-session/autostartutils.h]: good catch @palinek. Done.

**Comment by luis-pereira** [comment]
@palinek That's not the message anymore. Maybe you need to pull/rebase.
Check https://github.com/lxde/lxqt-session/commit/180b808957c7d79271633b8a70e7a1592bfaf5b7

**Pull request merged**